pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
I don't know if we should care about a site-wide default. The projects should all declare what versions they work with, and fail should there be an issue (that version isn't avaliable). At least that's my opinion. If you try to do something site-wide, adding a new version becomes tricky, because all of the sudden everyone gets it instead of those trying to upgrade. I think it would be better for companies that want to implement a site-wide version of this to have their own plugin that applies pygradle, then configures it for their need.